build/import/Packages.targets

```diff
@@ -29,7 +29,7 @@
     <PackageReference Update="RoslynTools.SignTool"                                                   Version="$(RoslynToolsSignToolVersion)" />
     <PackageReference Update="XliffTasks"                                                             Version="1.0.0-beta.20574.1" />
     <PackageReference Update="Microsoft.DiaSymReader.Pdb2Pdb"                                         Version="$(MicrosoftDiaSymReaderPdb2PdbVersion)" />
-    <PackageReference Update="Microsoft.Net.Compilers.Toolset"                                        Version="3.11.0" />
+    <PackageReference Update="Microsoft.Net.Compilers.Toolset"                                        Version="4.2.0-1.final" />
```

Comment:
Cant you use roslyn 4.1 stable for C# 10 as well?

Reply:
Yes you can.

I would like to propose setting `LangVersion` to `preview` in a future PR that also uses the new string syntax. Some of our unit tests could be tidied up.

Also we try to dogfood previews for extra testing.

Reply:
Maybe get these changes in first and try `preview` in another PR with the unit test cleanup?

Reply:
That's what I was thinking too.